ci_framework/roles/dlrn_promote/tasks/check_promotion_criteria.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Empty dlrnapi output (no jobs reported)
+  ansible.builtin.fail:
+    msg: "FAILURE: No successful jobs reported to DLRN API"
+  when:
+    - cifmw_dlrn_promote_hash_report_status_output.stdout | length == 0 or cifmw_dlrn_promote_hash_report_status_output.stdout == '[]'
+
+- name: Check for promotion criteria
+  when:
+    - cifmw_dlrn_promote_hash_report_status_output.stdout | length > 0
+    - cifmw_dlrn_promote_hash_report_status_output.stdout != '[]'
+  block:
+    - name: Fetch criteria file
+      ansible.builtin.fetch:
+        dest: "{{ cifmw_dlrn_promote_workspace }}/"
+        src: "{{ cifmw_dlrn_promote_criteria_file }}"
+        flat: true
+
+    - name: Load criteria for promotion
+      ansible.builtin.include_vars:
+        file: "{{ cifmw_dlrn_promote_workspace }}/{{ cifmw_dlrn_promote_criteria_file | basename }}"
+        name: "criteria"
```

Comment:
where does the criteria file for this live please

Reply:
Here is the same criteria file: https://github.com/openstack-k8s-operators/ci-framework/pull/558/files#diff-3aa519072c314b5764f735fe25431c00fadd1fb60f2f93f4a16ed06938a86d92 and actual promotion criteria file: https://review.rdoproject.org/r/c/rdo-jobs/+/50104/1/criteria/centos9/antelope.yaml

Reply:
thanks